bin/sce_qc_report.R

```diff
@@ -242,12 +242,16 @@ if (multiplexed) {
     table() |>
     as.list() # save as a list for json output
 
+  # for any ids that have no cells, set count to 0
+  missing_sample_ids <- setdiff(sample_ids, names(demux_counts))
+  demux_counts[missing_sample_ids] <- 0
+
```

Comment:
Your solution works, but a more direct way may be to have the `table()` work on a factor, with the ids already set. Then it will include zeros and preserve the order
```suggestion
    factor(levels = sample_ids) |> # use a factor to get zero counts
    table() |>
    as.list() # save as a list for json output
```